godtools/App/Features/AppLanguage/Presentation/DownloadableLanguages/DownloadableLanguagesView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  DownloadableLanguagesView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 12/4/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct DownloadableLanguagesView: View {
+    
+    @ObservedObject private var viewModel: DownloadableLanguagesViewModel
+    
+    init(viewModel: DownloadableLanguagesViewModel) {
+        
+        self.viewModel = viewModel
+    }
+    
+    var body: some View {
+        
+        VStack(spacing: 0) {
+                                    
+            SearchBarView(viewModel: viewModel.getSearchBarViewModel(), searchText: $viewModel.searchText)
+            
+            List {
+                ForEach(viewModel.downloadableLanguagesSearchResults) { downloadableLanguage in
+                    
+                    DownloadableLanguageItemView(downloadableLanguage: downloadableLanguage) {
+                        
+                        viewModel.downloadableLanguageTapped(downloadableLanguage: downloadableLanguage)
+                    }
+                    .listRowBackground(Color.clear)
+                }
+            }
+            .listStyle(.inset)
+            
+        }
+        .navigationBarBackButtonHidden(true)
+        .navigationTitle(viewModel.navTitle)
+        .environment(\.layoutDirection, ApplicationLayout.shared.layoutDirection)
```

Comment:
Nice got the environment modifier in.  I still need to see if there's a way we don't have to add these manually, it's going to be difficult to remember to add these.